gentoo-quick-installer.sh

```diff
@@ -148,27 +165,25 @@ echo "### Installing portage..."
 mkdir -p /etc/portage/repos.conf
 cp -f /usr/share/portage/config/repos.conf /etc/portage/repos.conf/gentoo.conf
 emerge-webrsync
+mv /etc/portage/make.conf /etc/portage/default
+mkdir /etc/portage/make.conf
+mv /etc/portage/default /etc/portage/make.conf
 
-echo "### Installing kernel sources..."
-
-emerge sys-kernel/gentoo-sources
+# required to allow for linux-firmware (required for binary kernel).
+echo "ACCEPT_LICENSE=\"*\"" >> /etc/portage/make.conf/default
 
 if [ "$USE_LIVECD_KERNEL" = 0 ]; then
```

Comment:
So I suggest to have something like GENTOO_BINARY=0/1. This flag will help to install in fastest way all binary. Otherwise standard kernel installation and package compilation will be used. What do you think? And USE_LIVECD_KERNEL can be deprecated with relevant code that copies kernel.

Reply:
This is a very interesting question. Is the purpose of the script to install packages as fast as possible? 

I would be inclined to say that the entire script should be focused on installing via binary, and that compilation should be avoided if at all possible. As [OSU's OSL's Stage 4](https://gentoo.osuosl.org/experimental/amd64/binpkg/default/linux/17.1/x86-64/) exists and is considered official, the only logical reason for source building is if you wanted to enable -march=native in the CFLAGS for that extra 1-2% performance.

If you really wanted to include a GENTOO_BINARY flag, probably best to just set it to 1 internally. Otherwise, I'd just omit the variable entirely, and set it. This issue might be out of the scope of this update, and I'll do a separate PR for binary strip. We can absolutely remove USE_LIVECD_KERNEL, though.

Reply:
Building kernel from the source can be needed if you want to build custom kernel module.
This CR removes completely this option, i would keep it, the rest is good.